test/Chainweb/Test/Pact/Checkpointer.hs

```diff
@@ -603,7 +603,7 @@ runSQLite' runTest sqlEnvIO = runTest $ do
     cp <- initRelationalCheckpointer initialBlockState sqlenv logger testVer testChainId
     return (cp, sqlenv)
   where
-    initialBlockState = set bsModuleNameFix True $ initBlockState $ genesisHeight testVer testChainId
+    initialBlockState = set bsModuleNameFix True $ initBlockState (1024^(2::Int)) $ genesisHeight testVer testChainId
```

Comment:
Could we make this a symbolic constant somewhere? We seem to refer to this 1 megabyte constant in a few places, but it's not clear from reading the test what it implies.